### PR TITLE
Set up landing index

### PR DESCRIPTION
**Pull request by marizaswart**
## What we have learnt
- The tasks in our PT needs to be clear and related to the tasks that we have to do to implement the user story and nothing extra
- Linking the restaurant is not possible as the restaurants are not created yet, so we could only add a placeholder untill that was done.

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 require 'coveralls'
-coveralls.wear_merged!('rails')
+Coveralls.wear_merged!('rails')
```
Comment (tochman) [inline features/support/env.rb]: This was fixed in another PR I think?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
-require ‘coveralls’
-Coveralls.wear_merged!(‘rails’)
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```
Comment (tochman) [inline spec/spec_helper.rb]: This was fixed in another PR? Do you have the latest code?

**Review by NegativeKarma — COMMENTED** [review]
LGTM

**Comment by NegativeKarma** [comment]
I used @diraulo style

File: features/landing_page.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Feature: Landing Page
+  As a visitor
+  In order to find food from a restaurant to order
+  I would like to see a list of restaurants when I visit the landing page
+
+  Background:
+    Given I visit the 'Landing' page
+
+  Scenario: Visitor visits landing page
+    Then I should see 'Welcome to our site!'
```
Comment (tochman) [inline features/landing_page.feature]: Try not using single quotes in your features.

**Comment by SophieSjoberg** [comment]
Good job! 👍

File: app/controllers/landing_controller.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```
Comment (tochman) [inline app/controllers/landing_controller.rb:1]: I don't mind this for now but you might want to point your `root_path` to another controller (that deals with restaurants) later.

**Review by tochman — COMMENTED** [review]
I don't think that this task is fulfilled: `add temporary Restaurant list with names linked to restaurant page`

**Comment by tochman** [comment]
I really need you to start ping-ponging the code. That should be reflected in the commit history.

File: spec/spec_helper.rb (hunk 2 above)
Comment (marizaswart) [reply spec/spec_helper.rb]: Fixed

**Comment by SophieSjoberg** [comment]
I only think we need to change the task on PT about the restaurant links, to make it clearer that restaurant titles should just be displayed for now. The rest LGTM

**Pull request merged**